authentication.md

```diff
@@ -47,14 +51,20 @@ GET /.well-known/openid-configuration
   ],
   "scopes_supported": [
     "openid",
-    "sub",
-    "aud",
-    "name",
-    "nickname",
+    "contacts:read",
+    "contacts:modify",
+    "conversations",
```

Comment:
If we think of this from the perspective of making a mobile app that replicates the site then I agree.  If this is being used by a different sort of app, let's say one that aggregates notifications only (maybe a bad example), then it could make sense to only ask for or grant read only permissions.

Reply:
I'm not sure this is something that will be used very often. In some rare cases this may be useful. For example to make an application that archives user data, including convrsations. But I don't really think it justifies have this read/modify separation of the conversations. And we can change this in the future versions of the API if we want. What do others think? @jhass @SuperTux88

Reply:
Let's keep it like this intially.